File: server/src/main/java/com/echolingo/server/model/VideoMeta.java
package com.echolingo.server.model;

public record VideoMeta(
        String videoId,
        String title,
        int duration,
        String thumbnailUrl,
        boolean hasDe,
        boolean hasEn,
        String deSource,
        String enSource,
        SubtitleScores scores,
        boolean cached,
        boolean ready
) {
}

File: android/app/src/main/java/com/echolingo/app/domain/model/VideoMeta.kt
package com.echolingo.app.domain.model

data class VideoMeta(
    val videoId: String,
    val title: String,
    val duration: Int,
    val thumbnailUrl: String,
    val hasDe: Boolean,
    val hasEn: Boolean,
    val scores: SubtitleScores,
)
